frontend/pages/projects/_id/custom/index.vue

```diff
@@ -0,0 +1,343 @@
+<template>
+  <layout-text v-if="doc.id" v-shortkey="shortKeys" @shortkey="changeSelectedLabel">
+    <template #header>
+      <toolbar-laptop
+        :doc-id="doc.id"
+        :enable-auto-labeling.sync="enableAutoLabeling"
+        :guideline-text="project.guideline"
+        :is-reviewd="doc.isConfirmed"
+        :total="docs.count"
+        class="d-none d-sm-block"
+        @click:clear-label="clear"
+        @click:review="confirm"
+      />
+      <toolbar-mobile :total="docs.count" class="d-flex d-sm-none" />
+    </template>
+    <template #content>
+      <v-card>
+        <v-card-title>
+          <label-group
+            :labels="categoryTypes"
+            :annotations="categories"
+            :single-label="exclusive"
+            @add="addCategory"
+            @remove="removeCategory"
+          />
```

Comment:
I think this is the code that handles the "categories" (the general label for a given text), is that correct? If so, I think the single-label props should be:
`:single-label="project.singleClassClassification"`

to follow the project's settings on creation.